setup.py

```diff
@@ -116,7 +116,8 @@ def girder_install(self, component):
             'girder-version.json',
             'conf/girder.dist.cfg',
             'mail_templates/*.mako',
-            'mail_templates/**/*.mako'
```

Comment:
I'm not sure we should get rid of this inclusion; in the future we may want to have subdirectories of mail templates and we'd want them to get sucked in automatically.